contracts/swap/swap.go

```diff
@@ -91,7 +90,6 @@ type Params struct {
 
 // ValidateCode checks that the on-chain code at address matches the expected swap
 // contract code.
-// TODO: have this as a package level function and pass the SimpleSwapBin as argument
```

Comment:
Removed. Does somebody object? Probably related: https://github.com/ethersphere/swarm/issues/1641

Reply:
Created a note in issue #1641